Tests/APITesters/AllAPITests/ObjcAPITester/RCVirtualCurrenciesAPI.m

```diff
@@ -0,0 +1,18 @@
+//
+//  RCVirtualCurrenciesAPI.m
+//  ObjcAPITester
+//
+//  Created by Will Taylor on 6/10/25.
+//
+
+@import RevenueCat;
+#import "RCVirtualCurrenciesAPI.h"
+
+@implementation RCVirtualCurrenciesAPI
+
++ (void)checkAPI {
+    RCVirtualCurrencies *virtualCurrencies = nil;
+    NSDictionary<NSString *, RCVirtualCurrency *> *all = virtualCurrencies.all;
```

Comment:
What about checking here the subscripting as well or wouldn't this work in Objective-C?